docs/source/development.rst

```diff
@@ -28,3 +28,53 @@ server and navigating to 0.0.0.0:8000 in your web browser.
 .. code::
 
     python3 -m http.server
+
+
+Changing the Theme
+==================
+Modifying CSS:
+
+**Do not** change CSS directly, instead change Post CSS files. This project is
+using Post CSS, specifically the `css-next <http://cssnext.io>`_ module. Post
+CSS allows us to have nested CSS selectors and makes it easier to read them.
+
+To Add a Post CSS file:
+
+1. Create a file called ``foo.pcss`` in ``jupyter_alabaster_theme/statuc/pcss``
+
+2. Add ``@mimport foo.css`` to ``index.css``.
+
+3. In ``package.json`` add a script with a name ``css:foo`` and action
+``"postcss jupyter_alabaster_theme/static/pcss/foo.pcss -o jupyter_alabaster_theme/static/css/foo.css",``
+
+4. Update ``css`` script in ``package.json`` to include ``npm run css:foo``.
+
+To compile Post CSS into CSS:
+
+.. code::
+
+    npm run css
+
+
+Modifying Sphinx templates:
+
+1. Make appropriate changes to templates.
+
+2. Uninstall ``jupyter_alabaster_theme``:
```

Comment:
I'm not exactly sure. When I did development locally, I found that just doing `pip install .` didn't change the package